gammapy/image/models/core.py

```diff
@@ -105,9 +122,10 @@ def __init__(self, lon_0, lat_0, sigma):
     def evaluate(lon, lat, lon_0, lat_0, sigma):
         """Evaluate the model (static function)."""
         sep = angular_separation(lon, lat, lon_0, lat_0)
-        norm = 1 / (2 * np.pi * sigma ** 2)
-        exponent = -0.5 * (sep / sigma) ** 2
-        return norm * np.exp(exponent)
+        a = 1.0 - np.cos(sigma)
+        norm = (1 / (4 * np.pi * a * (1.0 - np.exp(-1.0 / a)))) * u.sr ** -1
+        exponent = -0.5 * ((1 - np.cos(sep)) / a)
+        return (norm * np.exp(exponent)).to("deg-2")
```

Comment:
Instead creating the quantity and doing  the unit conversion here could you just create a `Quantity` object like so:
```
return u.Quantity(norm * np.exp(exponent), unit="sr1", copy=False)
```
The same as is done for the `SkyDisk` model...